apps/apple/FluentUITester/ios/FluentUITester.xcworkspace/contents.xcworkspacedata

```diff
@@ -2,7 +2,7 @@
 <Workspace
    version = "1.0">
    <FileRef
-      location = "group:FluentUIReactNativeTester_iOS.xcodeproj">
```

Comment:
I thought this file (contents.xcworkspacedata) was .gitignorable?

Reply:
I thought it could be but it's in the .gitignore and not being parsed out